monkestation/code/modules/particles/byond_particles/testing/turf.dm

```diff
@@ -0,0 +1,3 @@
+/turf/open/particle_test/Initialize(mapload)
+	. = ..()
+	add_emitter(/obj/emitter/fire, "fire")
```

Comment:
Testing turf still needed?

Reply:
its best to keep in incase we modify stuff in the future